.github/release-drafter.yml

```diff
@@ -0,0 +1,45 @@
+name-template: 'v$NEXT_MINOR_VERSION'
+tag-template: 'v$NEXT_MINOR_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - "feature"
+      - "enhancement"
+  - title: '💣 Breaking Change'
+    labels:
+      - "breaking"
+      - "change"
+  - title: '🐛 Bug Fixes'
+    labels:
+      - "fix"
+      - "bug"
+      - "bugfix"
+  - title: '📝 Documentation'
+    labels:
+      - "documentation"
+      - "docs"
+  - title: '🔨 Maintenance'
```

Comment:
do we really need those in the changelog?

Reply:
Do you mean the maintenance tasks? If so I don't think it hurts to have them, it's work that somebody has done 🙂